apis/addonconfigs/cni/v1alpha1/antrea-interworking_config_types.go

```diff
@@ -0,0 +1,176 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type AntreaInterworkingON struct {
```

Comment:
this crd is not needed, you only need to define the package, other than defining a new crd